server/views/applications/pages/health-needs/_health-needs-screen.njk

```diff
@@ -36,7 +36,7 @@
           }, {
             text: 'Learning difficulties and neurodiversity',
             href: paths.applications.pages.show({ id: applicationId, task: 'health-needs', page: 'learning-difficulties' }),
-            active: (pageName === 'learning-difficulties')
+            active: (pageName === 'learning-difficulties') or (pageName === 'learning-difficulties-details')
```

Comment:
could use an `includes` for this i.e. `['learning-difficulties','learning-difficulties-details'].includes(pageName)`